packages/ui-kit/src/design-system.js

```diff
@@ -14,8 +14,6 @@ export const pxToRem = (px, suffix = 'rem') => {
 
 export const colors = {
   light: {
-    // Used to give the website a unique tone (e.g. page titles, navigation links)
-    primary: customProperties.colorAccent40,
```

Comment:
To avoid confusion, we should remove it from here and always pick it from the theme.